build.js

```diff
@@ -533,10 +533,6 @@ function buildExtension(hook) {
 
                 const packageFileName = packageNameHyphen + '-' + packageParams.version + '.zip';
 
-                if (!fs.existsSync(cwd + '/build')) {
```

Comment:
This mkdir is needed as the function above may return earlier.

Reply:
Isn't the first mkdir blocked because it is mkdirSync?